pygments/lexers/markup.py

```diff
@@ -760,3 +763,69 @@ def _handle_cssblock(self, match):
     def __init__(self, **options):
         self.handlecodeblocks = get_bool_opt(options, 'handlecodeblocks', True)
         RegexLexer.__init__(self, **options)
+
+class TexinfoLexer(RegexLexer):
+    """
+    For `Texinfo <https://gnu.org/software/texinfo>`_ source code.
+
+    .. versionadded:: 2.10
+    """
+
+    name = 'Texinfo'
+    aliases = ['texi', 'texinfo']
+    filenames = ['*.texi']
+    mimetypes = ['text/x-texi']
+
+    tokens = {
+        'base': [
+            (r'\s+', Text),
+            (r'@$', Keyword),
+            (r'@[@:{}*]', Keyword),
+            (r'(?s)``.*?''', String),
+        ],
+        'commands': [
+            (r'(@[a-z]+)(\{)',
+             bygroups(Keyword, Punctuation), 'command'),
```

Comment:
Same here, not some kind of `Name`? Also, don't you want to recognize predefined builtins?